Main.java

```diff
@@ -9,7 +11,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Creates scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //gets first number from user
+    System.out.println("Please enter an integer:");
+    int a = input.nextInt();
+
+    // gets second number from user
+    System.out.println("Please enter another integer:");
+    int b = input.nextInt();
+
```

Comment:
Good work. Generally speaking, a and b would be poor choices for variable names. Remember, variables should always be named using camelCase and describe what they are storing or being used for.